xml/dns_versionbind.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  This fingerprint file matches the text string response from a DNS
+  version.bind request.
+
+  For example, the string 'dnsmasq-2.76-1-ubnt2' emitted by the command below:
+
+  $ nslookup -type=txt -class=chaos VERSION.BIND <dns_server> | grep VERSION.BIND | cut -d\" -f2
+  dnsmasq-2.76-1-ubnt2
+
+-->
+<fingerprints matches="dns_versionbind" protocol="dns" database_type="service" preference="0.750">
+  <!-- RedHat package naming: 
+      https://fedoraproject.org/wiki/Packaging:DistTag
+      https://fedoraproject.org/wiki/Packaging:Versioning
+  -->
+  <fingerprint pattern="^(9.[^-]+(?:-rpz\d?[+.]rl[\d.]+)?(?:-[SP]\d)?)-RedHat-[\d.]+[-.][\w.]+el([\d_]+)(?:.[\w.]+)?$">
+    <description>ISC BIND: RedHat Enterprise Linux</description>
+    <example service.version="9.8.2rc1">9.8.2rc1-RedHat-9.8.2-0.62.rc1.el6_9.2</example>
+    <example service.version="9.9.4">9.9.4-RedHat-9.9.4-38.el7_3.3</example>
+    <example service.version="9.3.6-P1" os.version="5_11">9.3.6-P1-RedHat-9.3.6-25.P1.el5_11.12</example>
```

Comment:
I don't believe there is a good way to handle this in recog, but shouldn't the version be 5.11, not 5_11?